c_src/stb_image_nif.c

```diff
@@ -234,23 +238,23 @@ static ERL_NIF_TERM to_file(ErlNifEnv *env, int argc, const ERL_NIF_TERM argv[])
         int stride_in_bytes = 0;
         int status = stbi_write_png(filename, w, h, comp, result.data, stride_in_bytes);
         if (!status) {
-            return error(env, "Unsuccesful attempt to write to png");
+            return error(env, "filed to write png");
```

Comment:
```suggestion
            return error(env, "failed to write png");
```

You have to fix the typo in other places. :)